templates/speech_recognition/LM/train.py

```diff
@@ -251,6 +252,11 @@ def text_pipeline(text):
         overrides=overrides,
     )
 
+    # We download the tokenizer from HuggingFace (or elsewhere depending on
+    # the path given in the YAML file).
+    run_on_main(hparams["pretrainer"].collect_files)
+    hparams["pretrainer"].load_collected(device=run_opts["device"])
```

Comment:
This was missing?! How can it be missing?!

Reply:
How about moving specific expected recipe structure to SB core modules?